scripts/update_schema_6.31.0.sql

```diff
@@ -0,0 +1,15 @@
+-- Add columns for metrics useful for BGE project.
+
+ALTER TABLE `iseq_product_metrics` \
+  ADD COLUMN `mean_bait_target_coverage` FLOAT(8,2) UNSIGNED DEFAULT NULL \
```

Comment:
As of MySQL 8.0.17, UNSIGNED for floats and FLOAT(M,D) expression are deprecated and will be removed https://dev.mysql.com/doc/refman/8.0/en/numeric-type-syntax.html

Reply:
```
ysql> ALTER TABLE `iseq_product_metrics` \
    ->   ADD COLUMN `mean_bait_target_coverage` FLOAT(8,2) UNSIGNED DEFAULT NULL \
    ->     COMMENT 'Mean coverage of the design target regions of a bait library (if used and known)' \
    ->     AFTER `on_or_near_bait_percent`, \
    ->   ADD COLUMN `target_autosome_filter` varchar(30) DEFAULT NULL \
    ->     COMMENT 'Filter used to produce the autosome target stats file' \
    ->     AFTER `target_percent_gt_coverage_threshold`, \
    ->   ADD COLUMN `target_autosome_length` bigint unsigned DEFAULT NULL \
    ->     COMMENT 'The total length of the autosomes only target regions' \
    ->     AFTER `target_autosome_filter`, \
    ->   ADD COLUMN `target_autosome_mapped_bases` bigint unsigned DEFAULT NULL \
    ->     COMMENT 'The number of mapped bases passing the autosome target filters' \
    ->     AFTER `target_autosome_length`;
Query OK, 0 rows affected, 2 warnings (4 min 29.17 sec)
Records: 0  Duplicates: 0  Warnings: 2

mysql> show warnings;
+---------+------+--------------------------------------------------------------------------------------------------------------------------+
| Level   | Code | Message                                                                                                                  |
+---------+------+--------------------------------------------------------------------------------------------------------------------------+
| Warning | 1681 | Specifying number of digits for floating point data types is deprecated and will be removed in a future release.         |
| Warning | 1681 | UNSIGNED for decimal and floating point data types is deprecated and support for it will be removed in a future release. |
+---------+------+--------------------------------------------------------------------------------------------------------------------------+
2 rows in set (0.00 sec)
```